api/api.py

```diff
@@ -1 +1,104 @@
-""""""
+'''API for the earthquake monitor.'''
+from datetime import datetime
+from flask import Flask, jsonify, request
+from database import (get_earthquake_by_id,
+                      get_earthquakes_by_magnitude,
+                      get_earthquakes_by_date,
+                      get_earthquakes_by_alert_level)
+
+app = Flask(__name__)
+
+
+def valid_color(color: str) -> bool:
+    """Checks whether the color is valid."""
+    return color in ('green', 'yellow', 'orange', 'red')
+
+
+@app.route("/", methods=["GET"])
+def endpoint_index():
+    """Return the enpoint for the API home page."""
+    return {"message": "Welcome to the Earthquake Monitor API homepage"}
+
+
+@app.route("/earthquakes/<int:earthquake_id>", methods=["GET"])
+def endpoint_get_earthquake(earthquake_id: int):
+    """Returns a specific earthquake"""
+    earthquake = get_earthquake_by_id(earthquake_id)
+
+    if not earthquake:
+        return jsonify({"error": "Earthquake not found"}), 404
+
+    return jsonify(earthquake), 200
+
+
+@app.route("/earthquakes/magnitude", methods=["GET"])
+def endpoint_get_magnitude():
```

Comment:
Possibly add error handling if someone tries making `min_magnitude` larger than `max_magnitude`?

Reply:
Also are there any boundaries to what value they can give?